plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -71,6 +74,17 @@ export class GraphQLInstrumentation extends InstrumentationBase {
       VERSION,
       Object.assign({}, DEFAULT_CONFIG, config)
     );
+    const ignoreMap = new Map();
+    config.ignoreFields?.map(item => ignoreMap.set(item, true));
```

Comment:
map is meant to return a new array if you want to iterate please use forEach instead

Reply:
you already created a new map, then you create a new new one and then you created a new private function to override the created at the beginning. 
why cant you simply do
```
config.ignoreFields?.forEach(item => this._ignoreMap.set(item, true));
```
?